### DISCUSSION TITLE
How to select working directory for cmake? "CMake: Select Active Folder" doesn't detect it

### DISCUSSION THREAD

**Discussion by Marqin** [microsoft/vscode-cmake-tools#3212] [Q&A] [answered]
My project structure is like this:
```
project-a
├── CMakeLists.txt
├── project-b
│   └── CMakeLists.txt
├── x
├── y
└── z
```

And I seem unable to select the CMake Tools extension to detect project-b.

When I use the "CMake: Select Active Folder" it lists only project-a and doesn't list project-b at all!

Is the only workaround to open project-b in separate VSCode window?

**Comment by bobbrow** [comment, marked as answer]
If you have two CMake projects in your workspace folder, you can specify both of them in the `cmake.sourceDirectory` in your `settings.json` file (VS Code settings) so that we know you want to work with both of them.

Example:
```
"cmake.sourceDirectory": [
  "/path/to/project-a",
  "/path/to/project-b"
]
```

You can also open both workspace folders in a single VS Code window if you'd prefer to do it that way.

**Reply by Marqin** [reply]
Thanks! That worked
